core/vibes/soul/sections/reviews/review-form.tsx

```diff
@@ -183,25 +183,25 @@ export const ReviewForm = ({
               value={typeof textControl.value === 'string' ? textControl.value : ''}
             />
             <Input
-              disabled={isAuthorDisabled}
```

Comment:
Forgot that `disabled` fields [don't submit with native HTML forms](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Attributes/disabled#:~:text=or%20even%20submitted%20with%20the%20form) 😅  This fixes that while still preventing modifications to the input.